persistent-postgresql/Database/Persist/Postgresql.hs

```diff
@@ -1261,19 +1275,27 @@ escape (DBName s) =
 data PostgresConf = PostgresConf
     { pgConnStr  :: ConnectionString
       -- ^ The connection string.
+    
+    , pgPoolStripes :: Int
```

Comment:
Adding new fields could be considered a breaking change for persistent-postgresql

Reply:
If they're exported, yeah, it's technically a breaking change.